core/src/main/java/org/modelmapper/internal/MappingEngineImpl.java

```diff
@@ -160,8 +160,12 @@ public MappingEngineImpl(InheritingConfiguration configuration) {
 
     if (condition != null) {
       if (!condition.applies(propertyContext)) {
-        context.shadePath(propertyPath);
-        return;
+        if (!mapping.isSkipped()) {
+          // Only shade the path when is not skipped
+          // Related to https://github.com/jhalterman/modelmapper/issues/197

```

Comment:
I like the change, but IMO the comments here aren't necessary since it's clear what the code is doing in this case (comments are good for complex things though) and the reference to the issue isn't really needed since Github keeps good history for the issue and any related commits :)